src/app/pages/registration/registration.component.ts

```diff
@@ -37,6 +37,7 @@ export class RegistrationComponent implements OnInit {
         this.router.navigate([this.returnUrl]);
       },
       error => {
```

Comment:
Remove this line

Reply:
@Manorlds-Eaglespark I have removed that, but kindly it is on the develop as well.